.github/workflows/k6-ci-loadtest.yml

```diff
@@ -94,3 +96,48 @@ jobs:
             --tag strategy=${{ steps.normalized.outputs.lt_strategy }}
             --tag run_id=${{ github.run_id }}
           inspect-flags: --include-system-env-vars
+
+  cron-test:
+    runs-on: arc-runners-arc-runner-set
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Normalize inputs
+        id: normalized
+        run: |
+          echo "lt_chain=${CHAIN:-Solana}" >> "$GITHUB_OUTPUT"
+          echo "lt_env=${ENV:-dev}" >> "$GITHUB_OUTPUT"
+          echo "lt_strategy=${STRATEGY:-constant_low_rate_1h}" >> "$GITHUB_OUTPUT"
```

Comment:
This will give us 1 request every 5 minutes, we want a bit more for Solana.
`constant_medium_rate_1h` should be fine for now (6 requests per minute)